app/uk/gov/hmrc/tariffclassificationfrontend/controllers/ReferCaseController.scala

```diff
@@ -35,18 +37,32 @@ class ReferCaseController @Inject()(authenticatedAction: AuthenticatedAction,
                                     val messagesApi: MessagesApi,
                                     implicit val appConfig: AppConfig) extends RenderCaseAction {
 
+  private val form: Form[Boolean] = MandatoryBooleanForm.form("refer_case")
+
   override protected val config: AppConfig = appConfig
   override protected val caseService: CasesService = casesService
 
   override protected def redirect: String => Call = routes.CaseController.applicationDetails
   override protected def isValidCase(c: Case)(implicit request: AuthenticatedRequest[_]): Boolean = c.status == OPEN
 
   def referCase(reference: String): Action[AnyContent] = authenticatedAction.async { implicit request =>
-    getCaseAndRenderView(reference, c => successful(views.html.refer_case(c)))
+    getCaseAndRenderView(reference, c => successful(views.html.refer_case(c, form)))
```

Comment:
this line could be extracted as 
```
 def showCase(reference: String, c: Case, f: Form[Boolean]) = {
     getCaseAndRenderView(reference, c => successful(views.html.refer_case(c, f)))
}
```
Then,
At line 49 you can call `showCase(reference, c, form)`
At line 56 you can call `showCase(reference, c, errors)`

Reply:
There is something similar in `CancelRulingController`

Reply:
As discussed I'm going to leave this as it's already readable